process-dataset/data-validation/data-prep.js

```diff
@@ -0,0 +1,42 @@
+const _ = require("lodash");
```

Comment:
The require is because I haven't set up babel in this repo. And yeah, JS is just a personal preference, you can use typescript with firebase (we do in the front end)